teerex/sgx-verify/src/collateral.rs

```diff
@@ -230,6 +230,9 @@ impl TcbInfo {
 	}
 
 	pub fn is_valid(&self, timestamp_millis: i64) -> bool {
+		log::info!("teerex: called into runtime call register_tcb_info(), inside Self::is_valid.");
+		log::info!("teerex: called into runtime call register_tcb_info(), inside Self::is_valid, self is: {:#?}" , &self);
+		log::info!("teerex: called into runtime call register_tcb_info(), inside Self::is_valid, timestamp_millis: {:#?}" , &timestamp_millis);
```

Comment:
Together with the above suggestion of setting a target instead of just having the `teerex` in the string I suggest:
```suggestion
		log::info!(target: TEEREX, "Called into runtime call register_tcb_info(), inside Self::is_valid.");
		log::info!(target: TEEREX, "self is: {:#?}" , &self);
		log::info!(target: TEEREX, "timestamp_millis: {:#?}" , &timestamp_millis);
```

What do you think?